Zivid/ReadIterateZDF/ReadIterateZDF.cpp

```diff
@@ -0,0 +1,59 @@
+/*
+This example shows how to import, display and iterate over a Zivid point cloud from a .ZDF
+file.
+*/
+
+#include <Zivid/CloudVisualizer.h>
+#include <Zivid/Zivid.h>
+
+#include <iostream>
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Setting up visualization" << std::endl;
+        Zivid::CloudVisualizer vis;
+        zivid.setDefaultComputeDevice(vis.computeDevice());
+
+        std::string Filename = "Zivid3D.zdf";
+        std::cout << "Reading " << Filename << " point cloud" << std::endl;
+        Zivid::Frame frame = Zivid::Frame(Filename);
```

Comment:
Can be `const`, and would be good to use the same convention for declaring variables which we use elsewhere:

```
const Zivid::Frame frame(Filename);
```

Reply:
done